backend/scripts/dev_run_background_jobs.py

```diff
@@ -75,7 +75,7 @@ def run_jobs() -> None:
     cmd_beat = [
         "celery",
         "-A",
-        "danswer.background.celery.versioned_apps.beat",
+        "danswer.background.celery.apps.beat",
```

Comment:
we should still use the versioned apps stub for consistency, but just point it at the single beat app.  Don't want someone to have to think about what entrypoint to use for the celery apps.